robocop/checkers/spacing.py

```diff
@@ -948,3 +965,31 @@ def parse_error(self, node, error):
                         col=indent + 1,
                     )
                     break
+
+
+class ArgumentsChecker(VisitorChecker):
+    reports = ("first-argument-in-new-line",)
+
+    def __init__(self):
+        super().__init__()
+
+    def visit_File(self, node):
```

Comment:
Parent class has visit_File that already does that - no point in overriding it if we don't add any extra code